src/appMain/CMakeLists.txt

```diff
@@ -212,7 +212,7 @@ endif ()
 # Install rules
 install(TARGETS ${PROJECT} DESTINATION bin)
 install(
-  FILES log4cxx.properties audio.8bit.wav test.txt smartDeviceLink.ini
+  FILES build_config.txt log4cxx.properties audio.8bit.wav test.txt smartDeviceLink.ini
```

Comment:
CMake fails the first time that this is run because the file doesn't exist at the time.

```
CMake Error at src/appMain/CMakeLists.txt:157 (file):
  file COPY cannot find
  "/mnt/hgfs/jacobkeeler/sdl_core/src/appMain/build_config.txt".
```